lib/neo-test-functions.js

```diff
@@ -600,7 +623,7 @@
     // Filter/Adapt particle
     // The elem to search start's counting at 1...
     const path = `particle.response.${typeToAdapt}.slice(${elementToSearch - 1}, ${elementToSearch})`
-    particle.response.content = eval(path)
+    particle.response[typeToAdapt] = eval(path)
```

Comment:
Same here: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_get

Would be cool, if we get rid of the evil `eval` 😉

Reply:
Same comment as above: https://github.com/neohelden/test-automation/pull/6#discussion_r618314536

https://github.com/neohelden/test-automation/pull/6/files#diff-f55cc924b5da9f0599b41b7520468148d530341b040fbba6b27cd7f14994055aR615